regulations/templates/regulations/preamble-partial.html

```diff
@@ -1,154 +1,153 @@
 {% load macros render_nested %}
 
-<section id="content-wrapper">
```

Comment:
Related to the above: `content-wrapper` is used by much of the JS. This seems like a very risky change; is it absolutely necessary?